### PR TITLE
Fix: logspam from missing mocks in onboarding integration tests

### PR DESCRIPTION
**Pull request by attfarhan**
Fixes https://github.com/sourcegraph/sourcegraph/issues/13456

### CODE REVIEW

File: web/src/integration/search-onboarding.test.ts [hunk 1]
```diff
@@ -5,6 +5,64 @@ import { commonWebGraphQlResults } from './graphQlResults'
...
+                    label: 'github.com/Algorilla/manta-ray',
+                    count: 1,
+                    limitHit: false,
+                    kind: 'repo',
+                },
+            ],
+            results: [
+                {
+                    __typename: 'Repository',
+                    id: 'UmVwb3NpdG9yeTozODcxOTM4Nw==',
```
Comment (felixfbecker) [inline web/src/integration/search-onboarding.test.ts:48]: **suggestion**: affording human editability in these stubs
```suggestion
                    id: 'Repository:Algorilla/manta-ray',
```

File: web/src/integration/search-onboarding.test.ts [hunk 2]
```diff
@@ -5,6 +5,64 @@ import { commonWebGraphQlResults } from './graphQlResults'
...
+                    name: 'github.com/Algorilla/manta-ray',
+                    label: {
+                        html:
+                            '\u003Cp\u003E\u003Ca href="/github.com/Algorilla/manta-ray" rel="nofollow"\u003Egithub.com/Algorilla/manta-ray\u003C/a\u003E\u003C/p\u003E\n',
```
Comment (felixfbecker) [inline web/src/integration/search-onboarding.test.ts:52]: **suggestion**: The `\u003C` and `\u003E` escape sequences make this look very cryptic and not human-editable and don't seem necessary. Can we replace them with `<` and `>` respectively? (in the whole file)

File: web/src/integration/search-onboarding.test.ts [hunk 3]
```diff
@@ -5,6 +5,64 @@ import { commonWebGraphQlResults } from './graphQlResults'
...
...RzLTEuMS0yLjQtMi40LTIuNHMtMi40LDEuMS0yLjQsMi40UzMzLjcsMjYuNCwzNSwyNi40eiIvPgoJPHBhdGggZD0iTTIzLDQyLjRjMS4zLDAsMi40LTEuMSwyLjQtMi40cy0xLjEtMi40LTIuNC0yLjRzLTIuNCwxLjEtMi40LDIuNFMyMS43LDQyLjQsMjMsNDIuNHoiLz4KCTxwYXRoIGQ9Ik01MCwxNmgtMS41Yy0wLjMsMC0wLjUsMC4yLTAuNSwwLjV2MzVjMCwwLjMtMC4yLDAuNS0wLjUsMC41aC0yN2MtMC41LDAtMS0wLjItMS40LTAuNmwtMC42LTAuNmMtMC4xLTAuMS0wLjEtMC4yLTAuMS0wLjQKCQljMC0wLjMsMC4yLTAuNSwwLjUtMC41SDQ0YzEuMSwwLDItMC45LDItMlYxMmMwLTEuMS0wLjktMi0yLTJIMTRjLTEuMSwwLTIsMC45LTIsMnYzNi4zYzAsMS4xLDAuNCwyLjEsMS4yLDIuOGwzLjEsMy4xCgkJYzEuMSwxLjEsMi43LDEuOCw0LjIsMS44SDUwYzEuMSwwLDItMC45LDItMlYxOEM1MiwxNi45LDUxLjEsMTYsNTAsMTZ6IE0xOSwyMGMwLTIuMiwxLjgtNCw0LTRjMS40LDAsMi44LDAuOCwzLjUsMgoJCWMxLjEsMS45LDAuNCw0LjMtMS41LDUuNFYzM2MxLTAuNiwyLjMtMC45LDQtMC45YzEsMCwyLTAuNSwyLjgtMS4zQzMyLjUsMzAsMzMsMjkuMSwzMywyOHYtMC42Yy0xLjItMC43LTItMi0yLTMuNQoJCWMwLTIuMiwxLjgtNCw0LTRjMi4yLDAsNCwxLjgsNCw0YzAsMS41LTAuOCwyLjctMiwzLjVoMGMtMC4xLDIuMS0wLjksNC40LTIuNSw2Yy0xLjYsMS42LTMuNCwyLjQtNS41LDIuNWMtMC44LDAtMS40LDAuMS0xLjksMC4zCgkJYy0wLjIsMC4xLTEsMC44LTEuMiwwLjlDMjYuNiwzOCwyNywzOC45LDI3LDQwYzAsMi4yLTEuOCw0LTQsNHMtNC0xLjgtNC00YzAtMS41LDAuOC0yLjcsMi0zLjRWMjMuNEMxOS44LDIyLjcsMTksMjEuNCwxOSwyMHoiLz4KPC9nPgo8L3N2Zz4K',
```
Comment (felixfbecker) [inline web/src/integration/search-onboarding.test.ts:56]: **suggestion**: to keep this one human-readable, perhaps we could put the SVG in a string constant and encode it on demand with `new Buffer(repositoryIconSvg).toString('base64')`?

File: web/src/integration/search-onboarding.test.ts [hunk 4]
```diff
@@ -21,6 +79,15 @@ describe('Search onboarding', () => {
...
             ...commonWebGraphQlResults,
+            SearchSuggestions: () => ({
+                search: {
+                    suggestions: [],
+                },
+            }),
+            Search: searchResults,
+            RepoGroups: () => ({
+                repoGroups: [],
+            }),
```
Comment (felixfbecker) [inline web/src/integration/search-onboarding.test.ts:90]: **question**: am I understanding correctly that `SearchSuggestions` and `RepoGroups` have no influence on the test, but `Search` does? Or are these results truthful and the backend would return no suggestions and repo groups?

File: web/src/integration/search-onboarding.test.ts [hunk 5]
```diff
@@ -5,6 +5,64 @@ import { commonWebGraphQlResults } from './graphQlResults'
 import { siteID, siteGQLID } from './jscontext'
 import assert from 'assert'
 import expect from 'expect'
+import { SearchResult } from '../graphql-operations'
+
+const searchResults = (): SearchResult => ({
```
Comment (felixfbecker) [inline web/src/integration/search-onboarding.test.ts:10]: **question**: can stubs here be shared with stubs used in search.test.ts? or should they be different?

**Review by felixfbecker — APPROVED** [review]
Thanks a lot!

File: web/src/integration/search-onboarding.test.ts (hunk 5 above)
Comment (attfarhan) [reply web/src/integration/search-onboarding.test.ts:10]: I actually took this from `search.test.ts`, could be shared.

File: web/src/integration/search-onboarding.test.ts (hunk 4 above)
Comment (attfarhan) [reply web/src/integration/search-onboarding.test.ts:90]: No influence from any of these, I'll make search return empty as well.

File: web/src/integration/search-onboarding.test.ts [hunk 6]
```diff
@@ -5,6 +5,27 @@ import { commonWebGraphQlResults } from './graphQlResults'
 import { siteID, siteGQLID } from './jscontext'
 import assert from 'assert'
 import expect from 'expect'
+import { SearchResult } from '../graphql-operations'
+
+const emptySearchResults = (): SearchResult => ({
```
Comment (felixfbecker) [inline web/src/integration/search-onboarding.test.ts:10]: **nitpick**: this is only referenced once, so could just be inlined?

**Review by felixfbecker — APPROVED** [review]

**Pull request merged**